Dictionary.WebApi/Services/ItemService.cs

```diff
@@ -52,5 +53,40 @@ public async Task CleanupAsync()
                     await _repository.DeleteItemsAsync(item.Id);
             }
         }
+
+        public async Task AppendItem(AppendItem itemDto)
+        {
+            var existingItem = await _repository.GetItemByKeyAsync(itemDto.Key!);
+
+            if (existingItem != null)
+            {
+                var contentList = JsonSerializer.Deserialize<List<object>>(existingItem.Content) ?? new List<object>();
+
+                contentList.Add(itemDto.ContentToAppend!);
+
+                existingItem.Content = JsonSerializer.Serialize(contentList);
+
+                existingItem.ExpirationPeriod = itemDto.ExpirationPeriod;
+                existingItem.ExpiresAt = DateTime.UtcNow.AddSeconds(itemDto.ExpirationPeriod ?? 0);
```

Comment:
if the expiration period is not given by the user we agreed to use the default value, not 0